application/web/index.html

```diff
@@ -95,6 +95,13 @@
                 <!-- Temporarily put textarea for the CodeMirror editor -->
                 <textarea class="entry-textarea" placeholder="Enter your thoughts here"></textarea>
                 <div class="past-entries"></div>
+
+                <div id="no-entry-text">
+                    <strong>You have no Journals</strong>
+                    <br>
```

Comment:
Instead of using multiple "\<br\>" tags you can use CSS to add padding/margins sparingly to create the spacing between the two blocks of test.

You can probably use query selector to grab the first <strong> element inside the no-entry-text div. and apply a bottom margin to it using CSS.